code/modules/surgery/organs/subtypes/xenos.dm

```diff
@@ -98,6 +98,8 @@
 			owner.adjustFireLoss(-heal_amt)
 			owner.adjustOxyLoss(-heal_amt)
 			owner.adjustCloneLoss(-heal_amt)
+	else//still should regen some plasma
+		owner.adjustPlasma(plasma_rate-5)//but at a lower rate
```

Comment:
This isn't a bugfix; this is a balance change.

I looked through our code prior to this, and Xenos never regenned plasma when not on weeds---with one sole exception: Hunters would regen 5 plasma every life process if they weren't running.